src/main/java/hudson/plugins/git/GitAPI.java

```diff
@@ -62,19 +48,24 @@ public GitAPI(String gitExe, FilePath repository, TaskListener listener, EnvVars
         this(gitExe, repository, listener, environment);
     }
 
+    @Deprecated
+    public GitAPI(String gitExe, FilePath repository, TaskListener listener, EnvVars environment) throws IOException, InterruptedException {
+        this(gitExe, repository, listener, environment, (Launcher) null);
```

Comment:
to disambiguate constructor call vs (deprecated) GitAPI(String, FilePath, TaskListener, EnvVars, String)